go.mod

```diff
@@ -221,6 +222,8 @@ require (
 
 replace github.com/pingcap/tidb-operator/pkg/apis => ./pkg/apis
 
+replace github.com/tikv/pd/client => github.com/HuSharp/pd/client v0.0.0-20231207064950-c060432d1ad9
```

Comment:
is it possible to remove this `replace` and just set the version in L49?

Reply:
yes we need to merge pd firstly